tableau/virtual_connection.go

```diff
@@ -7,6 +7,18 @@ import (
 )
 
 type VirtualConnection struct {
+	ID      string
+	Project struct {
+		ID string `json:"id,omitempty"`
+	} `json:"project,omitempty"`
+	Owner struct {
+		ID string `json:"id,omitempty"`
+	} `json:"owner,omitempty"`
+	Content string `json:"content,omitempty"`
+	Name    string `json:"name,omitempty"`
+}
+
+type ListedVirtualConnection struct {
```

Comment:
doesn the API actually have 2 types of object for this? They do look quite different but given most things are `omitempty` might be the same

Reply:
Yes, technically there are no issues on combining those as long as developers understand that asking list of virtual connections will populate ID, Name, CreatedAt, UpdatedAt, HasExtracts, IsCertified and WebPageURL fields and asking single virtual connection will populate  ID, Name, ProjectID, OwnerID and Content (ID and Name being only common fields in both scenarios).